administration/src/components/create-cards/CreateCardForm.tsx

```diff
@@ -35,48 +34,38 @@ const renderCardType: ItemRenderer<CardType> = (cardType, { handleClick, modifie
 
 interface Props {
   cardBlueprint: CardBlueprint
-  onUpdate: (newCardBlueprint: CardBlueprint | null) => void
+  onUpdate: () => void
+  onRemove: () => void
```

Comment:
I agree, that `onRemove` is a good idea, but I feel like your approach to `onUpdate` is not very react-like. I liked the previous version better.

Reply:
All of this is WIP.

The reason why the onUpdate changed is because it is not trivial to clone an class instance.

Blueprint is now a class instead of an interface.

Reply:
Ah I see... Yeah, then it's a bit trickier. I guess it's fine then 🤔